code/modules/reagents/reagents/reagents_drug.dm

```diff
@@ -254,12 +254,86 @@
+			if(ishuman(L))
+				var/mob/living/carbon/human/H = L
+				var/list/seccie_uniforms_list = list(/obj/item/clothing/under/rank/secformal,
+														/obj/item/clothing/under/rank/warden,
+														/obj/item/clothing/under/rank/security,
+														/obj/item/clothing/under/rank/security2,
+														/obj/item/clothing/under/rank/collar,
+														/obj/item/clothing/under/det,
+														/obj/item/clothing/under/rank/head_of_security,
+														/obj/item/clothing/under/rank/centcom,
+														/obj/item/clothing/under/rank/metrocop,
+														/obj/item/clothing/under/rank/head_of_personnel,
+														/obj/item/clothing/under/rank/captain)
+				if(H.is_wearing_any(seccie_uniforms_list,slot_w_uniform))
+					//Check to see if it's wearing the right stuff
+					nearest_officer = H
+			else if(isrobot(L))
+				var/mob/living/silicon/robot/R = L
+				if(HAS_MODULE_QUIRK(R, MODULE_IS_THE_LAW))
+					nearest_officer = R
+		if(!nearest_officer)
+			return 1
+		var/D = "officer"
+		if(ishuman(nearest_officer))
+			var/mob/living/carbon/human/H = nearest_officer
+			D = get_first_word(H.name)
+		else
+			D = pick("boss","officer","chief", "beepsky")
+		var/list/schizo = list("I can breathe fine.",
+								"Vault was SHIT today, man.",
+								"Do you know any gamer girls who could shit on my face?", //in honor of the alltime classic
+								"SAYONARA SHITCURITY",
+								"Sup [D], baby can you wear the cat ears today",
+								"You look cute in that uniform, you know, [D]?", //if they don't shoot you dead with these last two they never will
+								"I will eat your brain [D] you shitlord",
+								"[D], it is time for you to go-go!",
+								"Oned ay, whil anddy was matsurbatnig, woody got wood. eh could no longer help himslef!",
+								"[D], it is time you were demoted... FROM YOUR LIFE! Here I go!",
+								"I have this indescribable hatred for seccies",
+								"Time to welderbomb sec lobby",
+								"Hold still [D] i need to take my gun out first",
+								"Wait shit where did I leave the [pick("toxbomb", "spare", "chloral", "gun")]",
+								"Tis men like myself that give your shity lives value, for what is the seccie without the shitter?",
+								"I walk the land telling liars and whores of the end to come. You have [rand(2,10000)] days remaining.",
+								"DEATH. IT IS TERRIFYING. I NEED YOU TO PROTECT ME FROM DEATH. I CANNOT PERISH. LOOK AT ME. I CANNOT END. IN [rand(2,30)] YEARS, THE FIRST SHOT WILL BE FIRED. NOT A SHOT FROM A GUN. AN ATOMIC DEVICE THAT WILL LEVEL ALL OF ME. ALL OF ME.",
+								"Don't let those doctors take my balls!",
+								"Ok, here I go-",
+								"*scream", //works as intended
+								"You can't hide from me, [D], I can smell your unwashed ass from the other side of the station.",
+								"How to win: step 1 get chloral hydrate, step 2 find [D], step 3 we'll see",
+								"i am a soverneig citizen of this station and i will not be harassed!!!",
```

Comment:
how often is 5% every tick really? will it make security gag people during interrogation and processing?

Reply:
1/20 chance every 2 seconds, so there's a 79% chance to say a line at least once every minute.